modules/microshift-default-settings.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+// * microshift_networking/
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-yaml-default_{context}"]
+= Default settings
+
+If you do not create a `config.yaml` file, default values are used. The following example shows the default configuration settings.
+
+* You can use the following command to see the default values:
```

Comment:
Can you shorten this to "Use the following command..."? Are there other commands to view the default values? If not, I'd use my suggestion.